Add documentation for the Cielo Home integration (#42477)
home-assistant/home-assistant.github.io: source/_integrations/cielo_home.markdown

---
title: Cielo Home
description: Instructions on how to integrate Cielo smart thermostats in Home Assistant.
ha_category:
  - Climate
ha_iot_class: Cloud Polling

> ha_release: 2026.1

Reviewer: ```suggestion
ha_release: 2026.6
```

---
title: Cielo Home
description: Instructions on how to integrate Cielo smart thermostats in Home Assistant.
ha_category:
  - Climate
ha_iot_class: Cloud Polling

> ha_release: "2026.1"

Reviewer 2: ```suggestion
ha_release: 2026.1
```

ha_config_flow: true
ha_codeowners:
  - '@cielo-connect'
  - '@ihsan-cielo'

ha_domain: cielo_home
ha_platforms:
  - climate
ha_integration_type: integration
ha_quality_scale: bronze
---

The **Cielo Home** {% term integration %} integration allows you to control [Cielo](https://www.cielowigle.com) devices directly from Home Assistant, using the cloud-based **Cielo Connect API**.

## Supported devices

The **Cielo Home** {% term integration %} supports the following devices.

- **Cielo Breez Max**: Smart thermostat for mini-splits.
- **Cielo Breez Edge Pro**: Smart thermostat for mini-splits.
- **Cielo Thermostat Eco**: Smart thermostat for central HVAC systems.
- **Cielo Smart Thermostat**: Smart thermostat for central HVAC systems.

## Prerequisites

> Before integrating the Cielo device/s with Home Assistant, ensure that: 

Author: This has been resolved in commit:
https://github.com/home-assistant/home-assistant.io/pull/42477/commits/7d181c49103dd17a8a232673149f97c8512e133c

- Your Cielo device is already registered in the **Cielo Home** app. 

> - The device is in the supported devices.

Author: This has been resolved in commit:
https://github.com/home-assistant/home-assistant.io/pull/42477/commits/7d181c49103dd17a8a232673149f97c8512e133c

- **HVAC modes** (varies by device): Heat, Cool, Dry, Fan, Auto, Off
- **Temperature control**: Provide a target temperature where applicable.
- **Fan modes**: Provide a fan mode if applicable.
- **Vertical swing position**: Provide a vertical swing position where applicable.
- **Presets**: Home, Away, Sleep, & Pets (where supported & defined)

{% note %}

Some fan modes or swing positions depend on the device’s remote configuration.

{% endnote %}

#### Cielo Smart Thermostat & Cielo Smart Thermostat Eco

- **HVAC modes** (varies by equipment): Cool, Auto, Heat, Off
- **Temperature control**: Set the temperature setpoint.
- **Presets**: Home, Away, Sleep & pets (where supported and defined)

## Data updates

This integration relies on cloud {% term polling %}, with data refreshed approximately every 120 seconds (2 minutes). If the Cielo API or your device becomes unavailable, Home Assistant will automatically retry.

- Manual changes made in the Cielo Home app or via IR remote are synced automatically on the next poll.

{% note %}

If the device is added after the integration, you need to reload the Cielo Home integration.

{% endnote %}

## Actions

> This integration does _not_ provide any custom service actions. All controls are performed using the standard ‘Home Assistant’ climate services.

Reviewer 2: ```suggestion
This integration does _not_ provide any custom service actions. All controls are performed using the standard Home Assistant climate services.
```

---
title: Cielo Home
description: Instructions on how to integrate Cielo smart thermostats in Home Assistant.
ha_category:
  - Climate
ha_iot_class: Cloud Polling
ha_release: "2026.1"
ha_config_flow: true
ha_codeowners:
  - '@cielo-connect'
  - '@ihsan-cielo'

ha_domain: cielo_home
ha_platforms:
  - climate
ha_integration_type: integration
ha_quality_scale: bronze
---

> 

Reviewer 2: looking good! just one small thing I noticed. We don't use a line length limit. Please remove the line length limit and apply flowing text style.

Author: Resolved in new commit: https://github.com/home-assistant/home-assistant.io/pull/42477/commits/c68539359462d722cab5d503f5a0729a3807d8be

The **Cielo Home** {% term integration %} integration allows you to control [Cielo](https://www.cielowigle.com) 
devices directly from Home Assistant, using the cloud-based **Cielo Connect API**.

## Supported devices

The **Cielo Home** {% term integration %} supports the following devices.

- **Cielo Breez Max**: Smart thermostat for mini-splits.
- **Cielo Breez Edge Pro**: Smart thermostat for mini-splits.
- **Cielo Thermostat Eco**: Smart thermostat for central HVAC systems.
- **Cielo Smart Thermostat**: Smart thermostat for central HVAC systems.

## Prerequisites

Before integrating the Cielo device/s with Home Assistant, ensure that: 

- Your Cielo device is already registered in the **Cielo Home** app. 
- The device is in the supported devices.
- The device has a configured remote (automatically or manually). 
- Your Home Assistant server has internet access. 
- Before proceeding, ensure the Cielo device is fully compatible with Home Assistant.

> ### Important API Key Rules

Reviewer 2: ```suggestion
### Important API key rules
```

---
title: Cielo Home
description: Instructions on how to integrate Cielo smart thermostats in Home Assistant.
ha_category:
  - Climate
ha_iot_class: Cloud Polling
ha_release: "2026.1"
ha_config_flow: true
ha_codeowners:
  - '@cielo-connect'
  - '@ihsan-cielo'

ha_domain: cielo_home
ha_platforms:
  - climate
ha_integration_type: integration
ha_quality_scale: bronze
---

The **Cielo Home** {% term integration %} integration allows you to control [Cielo](https://www.cielowigle.com) 
devices directly from Home Assistant, using the cloud-based **Cielo Connect API**.

## Supported devices

The **Cielo Home** {% term integration %} supports the following devices.

- **Cielo Breez Max**: Smart thermostat for mini-splits.
- **Cielo Breez Edge Pro**: Smart thermostat for mini-splits.
- **Cielo Thermostat Eco**: Smart thermostat for central HVAC systems.
- **Cielo Smart Thermostat**: Smart thermostat for central HVAC systems.

## Prerequisites

Before integrating the Cielo device/s with Home Assistant, ensure that: 

- Your Cielo device is already registered in the **Cielo Home** app. 

> - The device is in supported devices 

Reviewer 2: ```suggestion
- The device is in the supported devices.
```

- The device has a configured remote (automatically or manually). 
- Your Home Assistant server has internet access. 
- Before proceeding, ensure the Cielo device is fully compatible with Home Assistant.

### Important API Key Rules

To safeguard security and prevent unauthorized access, Cielo enforces the following API security restrictions:

- Each API key can be used only once. 
- A new API key cannot be generated until the current one is revoked. 
- You may generate a maximum of 3 keys per month. 
- Once a new API key is generated, wait 60 seconds for it to be displayed on the screen.
- Revoking an API key will disconnect Home Assistant immediately or within about a minute.

### Obtaining an API key

**Retrieving API key:** `Device [name of device] not correctly registered with remote on Sensibo cloud.`

> 1. Visit the [Cielo Web Portal](https://home.cielowigle.com)

Reviewer 2: ```suggestion
1. Visit the [Cielo Web Portal](https://home.cielowigle.com).
```

> 2. Open the Menu by clicking the ☲ icon at the top left side.

Reviewer 2: ```suggestion
2. Open the Menu by selecting the ☲ icon at the top left side.
```
We prefer [select over click](https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/c/click)

3. Select **Home Assistant** from the sidebar.

> 4. The API key will be displayed. Copy the API Key from the dialog.

Reviewer 2: ```suggestion
4. The API key will be displayed. Copy the API key from the dialog.
```

{% tip %}

> If the message _This key is already used_ appears, you must first revoke the old key before generating a new one. 

Reviewer 2: ```suggestion
If the message **This key is already used** appears, you must first revoke the old key before generating a new one. 
```
I think this can be bold, as it is a UI string in that application.

{% endtip %}

{% include integrations/config_flow.md %}

If authentication succeeds, Home Assistant will automatically create the climate entities for all supported Cielo 
devices in your account.

{% note %}

If you add or set up a new Cielo device after configuring the integration in Home Assistant, you must reload the 
integration: 

1. Go to {% my integrations title="**Settings** > **Devices & services**" %}. 
2. Select **Cielo Home** and in the right corner, select the three dots {% icon "mdi:dots-vertical" %} menu.
3. Select **Reload**.

{% endnote %}

## Supported functionality

Each supported device exposes at least one **climate** entity.

{% note %}

Depending on device support, some entities might not be available as the device does not support them.

{% endnote %}

### Climates

The climate platform integrates the compatible Cielo devices into Home Assistant, allowing control through user 

> interface.

Reviewer 2: ```suggestion
The climate platform integrates the compatible Cielo devices into Home Assistant, allowing control through its user 
interface.
```

#### Cielo Breez Max & Cielo Breez Edge Pro

- **HVAC Modes (varies by device)**: Heat, Cool, Dry, Fan, Auto, Off
- **Temperature Control**: Provide a target temperature where applicable.
- **Fan Modes**: Provide a fan mode if applicable.
- **Vertical Swing Position**: Provide a vertical swing position where applicable.

> - **Presets**: Home, Away, Sleep, & Pets (where supported & defined)

Reviewer 2: ```suggestion
- **HVAC modes** (varies by device): Heat, Cool, Dry, Fan, Auto, Off
- **Temperature control**: Provide a target temperature where applicable.
- **Fan modes**: Provide a fan mode if applicable.
- **Vertical swing position**: Provide a vertical swing position where applicable.
- **Presets**: Home, Away, Sleep, & Pets (where supported & defined)
```
Are these UI strings? We use [sentence-style capitalization](https://learn.microsoft.com/en-us/style-guide/checklists/capitalization-checklist) also in UI strings.

{% note %}

Some fan modes or swing positions depend on the device’s remote configuration.

{% endnote %}

#### Cielo Smart Thermostat & Cielo Smart Thermostat Eco

- **HVAC Modes (varies by equipment)**: Cool, Auto, Heat, Off
- **Temperature Control**: Set the temperature setpoint.

> - **Presets**: Home, Away, Sleep & Pets (where supported and defined)

Reviewer 2: ```suggestion
- **HVAC modes** (varies by equipment): Cool, Auto, Heat, Off
- **Temperature control**: Set the temperature setpoint.
- **Presets**: Home, Away, Sleep & pets (where supported and defined)
```

## Data updates

> This integration relies on cloud polling, with data refreshed approximately every 120 seconds (2 minutes). If the Cielo API or your 

Reviewer 2: ```suggestion
This integration relies on cloud {% term polling %}, with data refreshed approximately every 120 seconds (2 minutes). If the Cielo API or your 
```
There's a [glossary term reference](https://developers.home-assistant.io/docs/documenting/integration-docs-examples#glossary-term-reference) for this term.

device becomes unavailable, Home Assistant will automatically retry.

- Manual changes made in the Cielo Home app or via IR remote are synced automatically on the next poll.

{% note %}

If the device is added after the integration, you need to reload the Cielo Home integration.

{% endnote %}

## Upcoming Features

Certain features are currently not supported by this integration. These include:

- Scheduling
- Advanced device features (such as Comfy Mode, Turbo, Internal Swing & LED controls)
- Full-state API commands

These capabilities will be added in future updates.

> 

Reviewer 2: ```suggestion
```
I would remove this. We don't usually include this section.

## Actions

This integration does _not_ provide any custom service actions. All controls are performed using the standard 
‘Home Assistant’ climate services.

## API limitations

- API key cannot be reused.

> - Revoking an API key will immediately disconnect Home Assistant

Reviewer 2: ```suggestion
- Revoking an API key will immediately disconnect Home Assistant.
```

- You can generate a maximum of three API keys within last 30 days.

## Troubleshooting

> This integration relies on an active internet connection and access to the Cielo Home API. Here’re few things to check 

Reviewer 2: ```suggestion
This integration relies on an active internet connection and access to the Cielo Home API. Here are a few things to check 
```

---
title: Cielo Home
description: Instructions on how to integrate Cielo smart thermostats in Home Assistant.
ha_category:
  - Climate
ha_iot_class: Cloud Polling
ha_release: "2026.1"
ha_config_flow: true
ha_codeowners:
  - '@cielo-connect'
  - '@ihsan-cielo'

ha_domain: cielo_home
ha_platforms:
  - climate
ha_integration_type: integration
ha_quality_scale: bronze
---

The **Cielo Home** {% term integration %} integration allows you to control [Cielo](https://www.cielowigle.com) 
devices directly from Home Assistant, using the cloud-based **Cielo Connect API**.

> To learn more about Cielo smart thermostats, visit [https://www.cielowigle.com](https://www.cielowigle.com).

Reviewer 2: ```suggestion
```
The link is provided in the intro. I don't think it is needed here.

> ## Supported Devices

Reviewer 2: ```suggestion
## Supported devices
```
We use [sentence-style capitalization](https://learn.microsoft.com/en-us/style-guide/capitalization), also in headings. Please fix throughout the page.

The **Cielo Home** {% term integration %} supports the following devices.

- **Cielo Breez Max**: Smart thermostat for mini-splits.
- **Cielo Breez Edge Pro**: Smart thermostat for mini-splits.
- **Cielo Thermostat Eco**: Smart thermostat for central HVAC systems.
- **Cielo Smart Thermostat**: Smart thermostat for central HVAC systems.

> ## Preliminary Steps

Reviewer 2: ```suggestion
## Prerequisites
```

Before integrating the Cielo device/s with Home Assistant, ensure that: 

> - Your Cielo device is already registered in the **Cielo Home app**. 

Reviewer 2: ```suggestion
- Your Cielo device is already registered in the **Cielo Home** app. 
```
Bold is mainly used to markup UI strings. Avoid using it to highlight content.

- The device is in supported devices 

> - The device has a **configured remote** (automatically or manually). 

Reviewer 2: ```suggestion
- The device has a configured remote (automatically or manually). 
```
Avoid using bold to highlight content. It is used to markup UI string. If you want to highlight text, you could use italics.

- Your Home Assistant server has internet access. 
- Before proceeding, ensure the Cielo device is fully compatible with Home Assistant.

> ## Important API Key Rules

Reviewer 2: ```suggestion
### Important API Key Rules
```

To safeguard security and prevent unauthorized access, Cielo enforces the following API security restrictions:

> - Each API key can be used **only once**. 

Reviewer 2: please remove bold markup.

Author: Bold Markups removed in new commit:
https://github.com/home-assistant/home-assistant.io/pull/42477/commits/439e7459770e9c9fea9a77839c2ef86e033b6da3

- A **new API key cannot be generated** until the current one is **revoked**. 
- You may generate a **maximum of 3 keys per month**. 
- Once a new API key is generated, wait 60 seconds for it to be displayed on the screen.
- Revoking an API key will disconnect Home Assistant immediately or within about a minute.

> ### Obtaining API Key

Reviewer 2: ```suggestion
### Obtaining an API key
```

**Retrieving API key:** `Device [name of device] not correctly registered with remote on Sensibo cloud.`

1. Visit the [Cielo Web Portal](https://home.cielowigle.com)
2. Open the Menu by clicking the ☲ icon at the top left side.
3. Select **Home Assistant** from the sidebar.
4. The **API key** will be displayed. **Copy** the **API Key** from the dialog.

{% tip %}
If the message “**This key is already used**” appears, you must first **revoke** the old key before generating a new one. 
{% endtip %}

{% include integrations/config_flow.md %}

If authentication succeeds, Home Assistant will automatically create the climate entities for all supported Cielo 
devices in your account.

{% note %}

If you add or set up a new Cielo device after configuring the integration in Home Assistant, you must reload the 
integration: 

Go to Settings → ‘Devices & Services’. Select **Cielo Home**. Open the overflow menu (⋮) in the right corner & select 

> ‘Reload’.

Reviewer 2: ```suggestion
1. Go to {% my integrations title="**Settings** > **Devices & services**" %}. 
2. Select **Cielo Home** and in the right corner, select the three dots {% icon "mdi:dots-vertical" %} menu.
3. Select **Reload**.
```
We have a [My link](https://developers.home-assistant.io/docs/documenting/integration-docs-examples#my-links) for the integration page.

{% endnote %}

## Supported functionality

Each supported device exposes at least one **climate** entity.

{% note %}

Depending on device support, some entities might not be available as the device does not support them.

{% endnote %}

### Climates

The climate platform integrates the compatible Cielo devices into Home Assistant, allowing control through user 
interface.

#### Cielo Breez Max & Cielo Breez Edge Pro

- **HVAC Modes (varies by device)**: Heat, Cool, Dry, Fan, Auto, Off
- **Temperature Control**: Provide a target temperature where applicable.
- **Fan Modes**: Provide a fan mode if applicable.
- **Vertical Swing Position**: Provide a vertical swing position where applicable.
- **Presets**: Home, Away, Sleep, & Pets (where supported & defined)

{% note %}

Some fan modes or swing positions depend on the device’s remote configuration.

{% endnote %}

#### Cielo Smart Thermostat & Cielo Smart Thermostat Eco

- **HVAC Modes (varies by equipment)**: Cool, Auto, Heat, Off
- **Temperature Control**: Set the temperature setpoint.
- **Presets**: Home, Away, Sleep & Pets (where supported and defined)

> ## Behind the Scenes

Reviewer 2: ```suggestion
## Data updates
```

- Data is refreshed every 60 seconds via the Cielo Cloud API.
- Manual changes made in the Cielo Home app or via IR remote are synced automatically on the next poll.

{% note %}

If the device is added after the integration, you need to reload the Cielo Home integration.

{% endnote %}

## Upcoming Features

Certain features are currently not supported by this integration. These include:

- Scheduling
- Advanced device features (such as Comfy Mode, Turbo, Internal Swing & LED controls)
- Full-state API commands

These capabilities will be added in future updates.

## Actions

This integration does **not** provide any custom service actions. All controls are performed using the standard 
‘Home Assistant’ climate services.

> ## Data Fetching and Limitations

Reviewer 2: Could this section be merged with the data updates section?

Author: Done! This section is now merged with data updates section, please see new commit:
https://github.com/home-assistant/home-assistant.io/pull/42477/commits/439e7459770e9c9fea9a77839c2ef86e033b6da3
